q2_feature_table/tests/test_summarize.py

```diff
@@ -383,6 +384,16 @@ def test_basic(self):
             index_fp = os.path.join(output_dir, 'index.html')
             self.assertTrue(os.path.exists(index_fp))
 
+            sample_frequency_fp = os.path.join(output_dir,
+                                               'sample-frequency-detail.html')
+            self.assertTrue(os.path.exists(sample_frequency_fp))
+
+            with open(sample_frequency_fp) as fi:
+                text = fi.read()
+                tbl_rx = re.compile(
+                    r'\{*[(S3)(S2)(S3)]*[(S3)(S2)(S3)]*[(S3)(S2)(S3)]*\}')
+                self.assertTrue(tbl_rx.search(text))
```

Comment:
On it!

Reply:
@Oddant1, @ebolyen updated, I used the built in ```json.loads()``` function because the ```pd.read_json()``` function did not work, but the ```loads()``` function worked fine. If necessary I could pass the results into a ```DataFrame``` constructor which should work fine.